app/src/main/java/com/eveningoutpost/dexdrip/cgm/connectfollow/Country.java

```diff
@@ -0,0 +1,152 @@
+package com.eveningoutpost.dexdrip.cgm.connectfollow;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum Country {
```

Comment:
Can you use `java.util.Locale` to get the country code?

Reply:
I didn't want  to get the country automatically, because the Medtronic user account is tied to the country. When logging into Medtronic CareLink at first the user must define the country to be used, so basically the Medtronic CareLink account is made up of 3 parameters: country, username, password.

Reply:
Then maybe just use an array of supported ISO 2-letter codes (i. e. `connectCountryValues`) and use the location API to get it translated into the country name. But please try to avoid reinventing the weel. ;-)

Reply:
Actually I don't need the names, just downloaded the list from another CareLink API and generated the enum to have a type. At first I thought there will be a different server for many countries, but at the end it turned out, that only US seems to have a sperate server and no the Country enum seems to be a little overhead. I will simplify it to the current requirements.

Reply:
Are the different countries still necessary or are there just two servers, on US server and one for the rest of the world (similar to the Dexcom Share servers)?

Reply:
Actually there are 3 + 1 servers: US, China (not implemented yet due to lack of test data), rest of the world + trial. There are country related rules, which could be important (for example MFA possible or MFA obligatory, but MFA is not supported by the current version). So currently the country is not so much important now, but it could become important later.